package.json

```diff
@@ -47,6 +47,6 @@
   },
   "peerDependencies": {
     "ethers": "^5.4.6",
-    "@zero-tech/zauction-sdk": ">=0.0.26"
+    "@zero-tech/zauction-sdk": ">=0.0.28"
```

Comment:
should we pin all dependencies as a rule? Obviously we control this one so could limit blast radius but ethers right there has a ^ on it. is there a way to get a linter or something to do this for us

Reply:
We do pin all dependencies as a rule, I just keep forgetting to do that. Would be great as a linter feature, I'll see if I can work that out

Reply:
Peer dependencies don't need to be pinned.